sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Added environment variable `AZURE_CLIENT_SEND_CERTIFICATE_CHAIN` support for `EnvironmentCredential`.
```

Comment:
@xiangyan99 Please also add a row to this table for the new env var: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/README.md#service-principal-with-certificate